src/components/Table/GridTable.tsx

```diff
@@ -1069,8 +1093,8 @@ type GridCollapseContextProps = {
 
 export const GridCollapseContext = React.createContext<GridCollapseContextProps>({
   headerCollapsed: false,
-  isCollapsed: (id: string) => false,
-  toggleCollapsed: (id: string) => {},
+  isCollapsed: () => false,
+  toggleCollapsed: () => {},
```

Comment:
question: I'm a little confused as to how you managed to remove the `id:string` args from here without any other code updates or breaking the build 🤔

Reply:
AH yeah, this is not the definition of `isCollapsed`, just the default impl, and functions are allowed to implement their spec w/o explicitly accepting all of the params if they don't use the params anyway.

Reply:
Right 🤦‍♂️

Reply:
...I need to remember that face palm emoji, I've needed it at least twice this morning and forgot what it was :-D